src/compiler/style/style-utils.ts

```diff
@@ -1,9 +1,3 @@
-import type * as d from '../../declarations';
-
-export const getStyleId = (cmp: d.ComponentCompilerMeta, modeName: string, isScopedStyles: boolean) => {
-  return `${cmp.tagName}${modeName}${isScopedStyles ? '.sc' : ''}`;
-};
-
 export const escapeCssForJs = (style: string) => {
```

Comment:
It looks like `escapeCssForJs` is only used in [one place](https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Aionic-team+escapeCssForJs), unit test file ([link](https://cs.github.com/ionic-team/stencil/blob/d134830b10cb8176b65e591fbb6198f844fff5e8/src/compiler/style/test/escape-css.spec.ts?q=org%3Aionic-team+escapeCssForJs)).  Can we remove this function (and its tests) while we're in here as well?

Reply:
I actually did that last night but forgot to push the change 🤪